src/constants/misc.js

```diff
@@ -2,6 +2,7 @@ export const DATA_URL = '/data/smartCitiesData.json';
 
 export const NO_CATEGORY = 'overview';
 export const NO_CITY = 'all-cities';
+export const ALL_CITIES = 'All Cities';
```

Comment:
A bit further down in this file we have a `STRINGS` bucket for all strings.

Reply:
Roger that.